### DISCUSSION TITLE
Does this program run on Google Collab too?

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [ThioJoe/YT-Spammer-Purge#102] [Q&A] [answered]
Hey,

Was wondering, if this program will run on Google Collab too, or if not.

**Comment by TechStudent10** [comment, marked as answer]
Just looked it up. I don't think so. Unless there is a way to import all the code files as well as the `client_secrets.json`, then yeah maybe it is possible. But if you're paying for Collab Pro, then maybe just switch to [Codespaces](https://github.com/codespaces). Honestly, I think it works better. #not-an-ad. Anyways, yeah you could try, but I don't think it will work. Maybe you should just wait for @ThioJoe's verdict.

**Reply by KendallDoesCoding** [reply]
> Just looked it up. I don't think so. Unless there is a way to import all the code files as well as the `client_secrets.json`, then yeah maybe it is possible. But if you're paying for Collab Pro, then maybe just switch to [Codespaces](https://github.com/codespaces). Honestly, I think it works better. #not-an-ad. Anyways, yeah you could try, but I don't think it will work. Maybe you should just wait for @ThioJoe's verdict.

Yeah, you can download the files from github, put it in your google drive and mount all the files inculding the .json one. Yeah, I'll check out Codespaces, thanks

**Comment by ThioJoe** [comment]
I'll look into it, it would certainly make things easier for people. Though I'm not sure how it work work with the GUI window part, I'd probably have to code up some kind of detection system and bypass that. Also would have to make sure the file access system works fine in the web UI.

**Reply by TechStudent10** [reply]
@ThioJoe @KendallDoesCoding you could use [Replit](https://replit.com). I **KNOW** it works from personal experience.

**Reply by KendallDoesCoding** [reply]
@ThioJoe I was just wondering if it would work on Google Collab, but even if it works on the programs that @TechStudent11 stated, it should work and be even better.

Google Collab has a lot of limitations to be honest, Idk about Replit or Codespaces, but if it can work in those platforms, that would work for me too.

**Reply by KendallDoesCoding** [reply]
Anyways, I will try to run it now on Collab

**Reply by KendallDoesCoding** [reply]
@TechStudent11 thanks, I tried it on replit and it worked, I even thought of a idea cause there were lots of complaints that it doesnt work on Linux. #118

**Reply by TechStudent10** [reply]
@KendallDoesCoding nice!

**Comment by KendallDoesCoding** [comment]
Ok, I'll check that out too.

Thanks
